src/segments/project.go

```diff
@@ -4,27 +4,43 @@ import (
 	"encoding/json"
 	"oh-my-posh/environment"
 	"oh-my-posh/properties"
+
+	"github.com/BurntSushi/toml"
 )
 
 type ProjectItem struct {
 	Name    string
 	File    string
-	Fetcher func(item ProjectItem) string
+	Fetcher func(item ProjectItem) (string, string)
+}
+
+type ProjectData struct {
+	Version string
+	Name    string
+}
+
+// Rust Cargo package
+type CargoTOML struct {
+	Package ProjectData
+}
+
+// Python Poetrey package
+type PyprojectTOML struct {
```

Comment:
I would respect the Pascal Casing: `PyProjectTOML`

Reply:
done